packages/nodes-base/credentials/SpotifyApi.credentials.ts

```diff
@@ -0,0 +1,24 @@
+import {
+	ICredentialType,
+	NodePropertyTypes,
+} from 'n8n-workflow';
+
+
+export class SpotifyApi implements ICredentialType {
+	name = 'spotifyApi';
+	displayName = 'Spotify Api';
```

Comment:
API should be caps? `API`?

Reply:
The codebase prefers `Api` for credentials. See [here](https://github.com/n8n-io/n8n/tree/master/packages/nodes-base/credentials).

Reply:
That's what I initially thought as well, but codebase convention is `Api`